lib/ash_authentication/strategies/password/actions.ex

```diff
@@ -279,4 +298,11 @@ defmodule AshAuthentication.Strategy.Password.Actions do
 
   def reset(strategy, _params, _options) when is_struct(strategy, Password),
     do: {:error, NoSuchAction.exception(resource: strategy.resource, action: :reset, type: :read)}
+
+  defp user_confirmed?(user, field) do
+    case Map.get(user, field) do
+      value when value in [nil, %Ash.NotLoaded{}, %Ash.ForbiddenField{}] -> false
```

Comment:
We can’t use `in` here because it does an exact equality filter. So we should match separately on each value.

Reply:
Thanks, I'll refactor this. I wanted to write a test for all these cases, but I do not know how to generate an user with those values for `:confirmed_at`. Any existing test I can look at for that ?